File: RestfulRent4Wheels/02 - Business Model/CarsModel.cs
using Microsoft.AspNetCore.Http;

namespace RentFourWheels
{
    public class CarsModel
    {
        public int Id { get; set; }
        public int TypeId { get; set; }
        public int Kilometrage { get; set; }
        public string Usability { get; set; }
        public string Availability { get; set; }
        public int Vin { get; set; }
        public int Branch { get; set; } // branch id 
        public IFormFile Image { get; set; }
        public string ImageFileName { get; set; }
    }
}

File: src/app/models/cars.model.ts
export class carsModel {

    constructor (
        public id?: number, 
        public carTypeId?: number, 
        public manufacturer?: string,
        public model?: string,
        public yearOfManufacture?: number,
        public gearBox?: string,
        public name?: string,
        public kilometrage?: number, 
        public usability?: string,
        public availability?: string,
        public vin?: number,
        public branchName?: number,
        public image?: File,
        public imageFileName?: string,
        ) {}
}